recipes/gwsurrogate/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "gwsurrogate" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 410085805cd07aa7c009c3f7b7313dcb5eec57184cc20ad663f210cf04fc8f0e
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - gsl
+    - gwtools
+    - numpy
```

Comment:
If you build against numpy, you should follow these instructions: https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy

Reply:
@scopatz, I have a `pin_compatible()` in the right place in the `run` list (AFAICT). What else is missing? Sorry if I'm missing something obvious.